cla_backend/apps/reports/management/commands/mi_cb1_report.py

```diff
@@ -0,0 +1,29 @@
+# coding=utf-8
+import logging
+from datetime import date, timedelta
+from django.core.management.base import BaseCommand
+from reports.tasks import ExportTask
+from core.models import get_web_user
+from django.views.decorators.csrf import csrf_exempt
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "This runs the MCCB1sSLA report"
+
+    def handle(self, *args, **options):
+        self.create_report()
+
+    @csrf_exempt
+    def create_report(self):
+        date_from = (date.today() - timedelta(days=7)).strftime("%Y-%m-%d")
+        date_to = date.today().strftime("%Y-%m-%d")
+
+        report_data = '{{"action": "Export", "csrfmiddlewaretoken": "PQk4Pt55CL0NBapx9hSqZTJkSn6tL6TL", "date_from": "{0}", "date_to": "{1}"}}'.format(
```

Comment:
Do we need this hardcoded `csrfmiddlewaretoken`?

Reply:
this has been removed.